tests/core/test_views.py

```diff
@@ -529,6 +538,64 @@ def test_update_profile(self):
 
         self.assertNotEqual(test_auth, None)
         self.assertEqual(test_auth.email, 'test@test.com')
+        messages = list(response.context['messages'])
+        self.assertEqual(len(messages), 1)
+        self.assertEqual(messages[0].message, gettext("Account info saved!"))
+        self.assertEqual(messages[0].level, msg_constants.INFO)
+
+
+@wiki_override_settings(ACCOUNT_HANDLING=True)
+class LogoutViewTests(RequireRootArticleMixin, DjangoClientTestBase):
+
+    def test_logout_account_handling(self):
+        response = self.client.get(wiki_settings.LOGOUT_URL)
+        messages = list(response.wsgi_request._messages)
+        user = auth.get_user(self.client)
+        self.assertIs(auth.get_user(self.client).is_authenticated(), False)
+        self.assertIsInstance(user, AnonymousUser)
+        self.assertEqual(len(messages), 1)
+        self.assertEqual(messages[0].message, gettext("You are no longer logged in. Bye bye!"))
```

Comment:
I think this line can be safely removed -- it might be a personal preference? But I think tests should not block that things change and improve, but should test that things don't break. So the essential test is to see that the user was actually logged out as in `self.assertIsInstance(user, AnonymousUser)`.

Reply:
OK. I can remove it. It took me a little while to figure out the messages thing in testing.

Reply:
I don't agree that these things are unimportant to test. They should remain static, so in essence, it's nice to have such things in the test suite. Anyway, I removed them consistently.

Reply:
I've tried many times that fixing things become an extra burden, because tests have been written so statically that they break for just about anything. I think this test is really fine, but checking the natural language is always a bit too specific for my own personal taste. It's hard to have a rapid development of a project when everything gets too locked down in test specifications. But this is probably a long discussion :)